docs/User-Guide.md

```diff
@@ -81,37 +94,77 @@ usage:
+                        choose if it should return regular text or other forms
                         of output
-  --log [log-level]:[number-of-lines]
-  --bot [start|stop|restart|status]
-  --botnet [start|stop|restart|status]
-  --list [bots|queues]
+  --log [log-level]:[number-of-lines], -l [log-level]:[number-of-lines]
+                        Reads the last lines from bot log, or from system log
+                        if no bot ID was given. Log level should be one of
+                        DEBUG, INFO, ERROR or CRTICAL. Default is INFO. Number
+                        of lines defaults to 10, -1 gives all. Reading from
+                        system log is not implemented yet.
+  --bot [start|stop|restart|status], -b [start|stop|restart|status]
+  --botnet [start|stop|restart|status], -n [start|stop|restart|status]
+  --list [bots|queues], -s [bots|queues]
+  --clear queue, -c queue
+                        Clears the given queue in broker
 
-description: intelmqctl is the tool to control intelmq system
+description: intelmqctl is the tool to control intelmq system. Outputs are
+logged to /opt/intelmq/var/log/intelmqctl
 ```
 
+### Botnet
+
+The botnet represents all currently configured bots. To get an overview which
+bots are running, use `intelmqctl -n status`.
 
-### Utilities
+## Utilities
 
-#### Monitoring Logs
+### Monitoring Logs
+
+All bots and `intelmqctl` log to `/opt/intelmq/var/log/`. In case of failures,
+messages are dumped to the same directory with file ending `.dump`.
 
 ```
-$ tail -f /opt/intelmq/var/log/*
+$ tail -f /opt/intelmq/var/log/*.log
 ```
 
-#### Reset Pipeline and Cache (be careful)
+### Reset Pipeline and Cache (be careful)
 
 ```
 $ redis-cli FLUSHDB
 $ redis-cli FLUSHALL
 ```
 
+<a name="configuration"></a>
+# Configuration
+
+By default, one collector, one parser and one output are started. The default
+collector an parser handle data from malware domain list, the file output bot
+writes all data to `/opt/intelmq/var/lib/bots/file-output/events.txt`.
+
+The configuration directory is `/opt/intelmq/etc/`, all files are JSON.
+
+* `defaults.conf`: Some default values for bots and their behavior, e.g.
+error handling, log options and pipeline configuration. Will be removed in
+[future](https://github.com/certtools/intelmq/issues/267).
+* `system.conf`: System configuration for e.g. the logger and the pipeline.
+* `startup.conf`: Maps the bot ids to python modules.
+* `runtime.conf`: Configuration for the individual bots.
+* `pipeline.conf`: Defines source and destination queues per bot.
+* `BOTS`: Includes configuration hints for all bots. E.g. feed URLs or
+database connection parameters. Use this as a template for `startup.conf`
+and `runtime.conf`. Also read by the intelmq-manager.
+
+To configure a new BOT, you need to define it first in `startup.conf`.
+Then do the configuration in `runtime.conf` using the bot if.
+Configure source and destination queues in `pipeline.conf`.
```

Comment:
here we can add a note mentioning the intelmq-manager to help generating the configs needed